docs/pallas/pipelining.md

```diff
@@ -297,18 +297,19 @@ def double_buffered_pipeline(
   for i in range(grid_size):
     cur_slot = i % 2
     next_slot = (i + 1) % 2
-    if i < grid_size:
-      copy_in_start(in_hbm[data_slices(i+1)], in_sram[next_slot])
+    if (i + 1) < grid_size:
+      copy_in_start(in_hbm[in_slices(i+1)], in_sram[next_slot])
     copy_in_wait(in_sram[cur_slot])
 
-    kernel(inputs, outputs)
+    kernel(in_sram[cur_slot], in_sram[next_slot])
```

Comment:
This should be `kernel(in_sram[cur_slot], out_sram[cur_slot])`

Reply:
Oh yes! Missed it! Thanks for catching that!